backend/game_handler.py

```diff
@@ -64,8 +130,15 @@ def get_multiplicator(self) -> float:
             )
             return multiplicator
 
+    def cashout(self, ws):
+        if ws in self.bids:
+            gain = self.get_multiplicator() * self.bids[ws]
+            self.bids.pop(ws, None)
+            self.game_handler_parent.cash[ws] += gain
+            return gain
+
     def is_waiting(self) -> bool:
-        return (time.time() - self.waiting_initial_time) > 30  # 30 seconds wait for now
+        return (time.time() - self.waiting_initial_time) < 10  # 30 seconds wait for now
```

Comment:
```suggestion
        return (time.time() - self.waiting_initial_time) < 10  # 10 seconds wait for now
```
Update comment to reflect change in behaviour